### ISSUE TITLE
should interface AuthProviderCallback accept null on accessToken?

### ISSUE THREAD

**Issue by sibelius** [microsoftgraph/msgraph-sdk-javascript#115]
https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/0f9ebecb26b0ba99210ba0bae8e739cafcbf4b5a/src/common.ts#L12-L14

it makes sense when we got an error to get an access token

**Comment by muthurathinam** [comment]
@sibelius Yeah this does make sense, and thanks for bringing this to light.
Will make the change. If you have done it first, submit a pull request. Am happy to review !
